src/python/CMSSpark/reports/visualize_campaigns.py

```diff
@@ -0,0 +1,213 @@
+import pandas as pd
+
+import matplotlib as mpl
+# We will not be showing images because we don't haw UI
+mpl.use('Agg')
+import matplotlib.pyplot as plt
+from subprocess import check_output
+from math import log
+import os
+import shutil
+import operator
+import argparse
+
+PLOTS_PATH = '../CERNTasks.wiki/images/campaign_plots/'
+
+report = ''
+
+def append_report(lines):
+    global report
```

Comment:
remove **all** globals and never use them, they always lead to hard maintenance in a code base